CefSharp.Example/CefExample.cs

```diff
@@ -95,6 +95,9 @@ public static void Init(AbstractCefSettings settings, IBrowserProcessHandler bro
             //Possibly useful when experiencing blury fonts.
             //settings.CefCommandLineArgs.Add("disable-direct-write", "1");
 
+            //settings.CefCommandLineArgs.Add("force-renderer-accessibility", "1");
```

Comment:
Please add a comment/link as to what this command line argument does (or remove it if it's not relevant).